.github/workflows/tvos-build.yml

```diff
@@ -83,4 +85,4 @@ jobs:
 
       - name: Build app
         working-directory: ${{ env.WORKING_DIRECTORY }}
-        run: yarn react-native run-ios --no-packager --scheme TVOSExample-tvOS --simulator "Apple TV"
+        run: yarn react-native-tvos run-ios --no-packager --scheme TVOSExample-tvOS --simulator "Apple TV"
```

Comment:
Q: Why was this change necessary?

Reply:
There's no package named `react-native` in this catalogue. I haven't checked why is that so, but `react-natvie-tvos` forks `react-native`'s CLI so it's what we want to use there.

Reply:
good, thanks!